src/common/coursesnotfound/__tests__/CoursesNotFound.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { MemoryRouter } from 'react-router';
+
+import { renderWithRouter } from 'lib/utils/jest/index';
+import { getCurrentTerm, getReadableTerm } from 'lib/utils/terms';
+
+import { CoursesNotFound } from '../CoursesNotFound';
+
+const term = getCurrentTerm();
+it('is in the document', () => {
+  renderWithRouter(<CoursesNotFound />, { route: `/${getCurrentTerm()}` });
```

Comment:
The reason why this doesn't work is that when you render this component, you end up having no `term` route parameter encapsulating it. 

Normally when you use the `useParams` and such, its within a `Route` component with the route params defined with something like `/:term` (look in `routes.tsx`). If you want to test this with the term showing up as expected, you need to wrap it with a `Route` component.

Alternatively you make this component not reliant on `useParams` and inject it into the page as props.